knative-operator/deploy/resources/console_cli_download_kn_deployment.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: serverless-operator
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: serverless-operator
+  name: kn-cli-downloads
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: serverless-operator
+      component: kn-cli-downloads
+  template:
+    metadata:
+      name: kn-cli-downloads
+      labels:
+        app: serverless-operator
+        component: kn-cli-downloads
+    spec:
+      containers:
+      - name: kn-download-server
+        terminationMessagePolicy: FallbackToLogsOnError
+        image: quay.io/openshift-knative/kn-cli-artifacts:latest
```

Comment:
Should this ever pull latest? I'd expect it to be pinned to the specific `kn` version that directly matches this release of the operator. And we'll have to ensure automation replaces this with the corresponding actual product build from registry.redhat.io instead of quay.io.

Reply:
Nope, not latest. It was added for testing (even before we cut any tagged release of this image).
https://github.com/openshift-knative/serverless-operator/pull/178#discussion_r392846395

Reply:
Added tagged version for now from openshift registry.